hazelcast/src/main/java/com/hazelcast/internal/ascii/memcache/IncrementCommandProcessor.java

```diff
@@ -68,31 +71,23 @@ public void handle(IncrementCommand incrementCommand) {
 
     private void incrementUnderLock(IncrementCommand incrementCommand, String key, String mapName) {
         Object value = textCommandService.get(mapName, key);
-        MemcacheEntry entry;
 
         if (value != null) {
-            if (value instanceof MemcacheEntry) {
-                entry = (MemcacheEntry) value;
-            } else if (value instanceof byte[]) {
-                entry = new MemcacheEntry(incrementCommand.getKey(), (byte[]) value, 0);
-            } else if (value instanceof String) {
-                entry = new MemcacheEntry(incrementCommand.getKey(), stringToBytes((String) value), 0);
-            } else {
-                try {
-                    entry = new MemcacheEntry(incrementCommand.getKey(), textCommandService.toByteArray(value), 0);
-                } catch (Exception e) {
-                    throw ExceptionUtil.rethrow(e);
-                }
-            }
+            MemcacheEntry entry = entryConverter.toEntry(incrementCommand.getKey(), value);
 
-            final byte[] value1 = entry.getValue();
-            final long current = (value1 == null || value1.length == 0) ? 0 : byteArrayToLong(value1);
-            long result = -1;
-            result = incrementCommandTypeCheck(incrementCommand, result, current);
+            String currentCachedValue = bytesToString(entry.getValue());
 
-            incrementCommand.setResponse(concatenate(stringToBytes(String.valueOf(result)), RETURN));
-            MemcacheEntry newValue = new MemcacheEntry(key, longToByteArray(result), entry.getFlag());
```

Comment:
Can we delete `longToByteArray` methods? It looks like there are no usage left.

Reply:
Yes, we can. I'll do that.

Reply:
[Done](https://github.com/hazelcast/hazelcast/pull/19653/commits/60dfe60edede117803f35dec5c117e151e101b35#diff-f994c6ca2d7177aa4fda391bba69665b94aa22b206aa38db6e0fd695edc09a1c).